src/rules/BackgroundAbsoluteURLRule.js

```diff
@@ -0,0 +1,14 @@
+// This is a very simple test, and could be made more robust.
+// All we do is check for the presence of "http://" or "https://"
+// at the start of the string.
+
+const description = 'Relative background URLs can break (i.e. if recipients are outside the company network) and make your content unavailable to view.';
+
+
+export default (props) => {
+  if (props.hasOwnProperty('background')) {
+    if (!/^https?:\/\//.test(props['background'])) {
```

Comment:
The form for the background URL in the shorthand is one of:

url(...)
url('...')
url("...")

Checking if the background property starts with "http://" or "https://" won't be enough. Perhaps we can use something like this. It's not the most robust since it assumes the url will be wrapped (or not wrapped) by matching quotes.

```js
/url\(['"]?https?:\/\/.+['"]?\)/
```